src/plugin.js

```diff
@@ -214,9 +214,11 @@ const onPlayerReady = (player, emeError) => {
   setupSessions(player);
 
   if (window.WebKitMediaKeys) {
-    // Support Safari EME with FairPlay
-    // (also used in early Chrome or Chrome with EME disabled flag)
-    player.tech_.el_.addEventListener('webkitneedkey', (event) => {
+    player.eme.webkitneedkey = {
```

Comment:
Make it pseudo-private
```suggestion
    player.eme.webkitneedkey_ = {
```